drivers/hmis/spec/requests/hmis/external_form_submissions_spec.rb

```diff
@@ -57,20 +56,66 @@
   end
 
   it 'should resolve external form submissions' do
-    submission = create(:hmis_external_form_submission, definition: form_definition, submitted_at: today.midnight)
-    submission.process_custom_data_elements!(form_definition: form_definition)
+    submission = create(:hmis_external_form_submission, definition: definition, submitted_at: today.midnight)
     filters = { 'status' => 'new', submitted_date: today.strftime('%Y-%m-%d') }
     variables = {
       id: p1.id,
-      formDefinitionIdentifier: form_definition.identifier,
+      formDefinitionIdentifier: definition.identifier,
       filters: filters,
     }
     response, result = post_graphql(variables) { query }
-    expect(response.status).to eq 200
+    expect(response.status).to eq(200), result.inspect
     expected = {
       'id' => submission.id.to_s,
-      'customDataElements' => [{ 'key' => 'your_name' }],
+      'values' => { 'your_name' => 'value' },
     }
     expect(result.dig('data', 'project', 'externalFormSubmissions', 'nodes')).to contain_exactly(expected)
   end
+
+  context 'when form rule applies to the organization, not the project' do
+    let!(:definition) { create :hmis_external_form_definition }
+    let!(:rule) { create :hmis_form_instance, definition_identifier: definition.identifier, entity: p1.organization, active: true }
+
+    it 'should return submissions' do
+      submission = create(:hmis_external_form_submission, definition: definition, submitted_at: today.midnight, raw_data: { 'your_name' => 'ebeneezer' })
+      response, result = post_graphql({ id: p1.id, formDefinitionIdentifier: definition.identifier }) { query }
+      expect(response.status).to eq(200), result.inspect
+      expected = {
+        'id' => submission.id.to_s,
+        'values' => { 'your_name' => 'ebeneezer' },
+      }
+      expect(result.dig('data', 'project', 'externalFormSubmissions', 'nodes')).to contain_exactly(expected)
+    end
+  end
+
+  context 'when there are several external forms' do
+    let!(:definition_2) do
+      fd = create(:hmis_external_form_definition)
+      HmisExternalApis::PublishExternalFormsJob.new.perform(fd.id)
+      fd.reload
```

Comment:
Does the form need to be run through this job before testing? I think the factory should generate the CDEDs for it, so this might not be necessary anymore. (I see it was present in the older test case above).